File: src/Components/Contrato.js
import '../styles.css';
export default function Contrato ({contrato}){
    return(
        <div className='contrato-container'>
            <h4>
                {contrato.Nombre}
            </h4>
            <p>Fecha: {contrato.Fecha.toLocaleDateString()}</p>
            <p>Monto: {contrato.Monto}</p>
            <p>IdCliente: {contrato.IdCliente}</p>
        </div>
    )
}

File: api/api/Models/Contrato.cs
namespace api.Models
{
  public class Contrato
  {
    public int Id { get; set; }
    public int ClienteId { get; set; }
    public string? Nombre { get; set; }
    public float monto { get; set; }
    public DateTime Fecha { get; set; }
  }
}
